splinterd/src/main.rs

```diff
@@ -25,8 +25,11 @@ mod error;
 mod routes;
 mod transport;
 
-use flexi_logger::{style, DeferredNow, LogSpecBuilder, Logger};
+use flexi_logger::{style, DeferredNow};
```

Comment:
flexi_logger should be completely disabled when log4rs is enabled.

Reply:
fixed